src/xdocs/cmdline.xml.vm

```diff
@@ -52,7 +52,7 @@ java -D&lt;property&gt;=&lt;value&gt;  \
         checks to apply. Command line options are:
       </p>
 
-      <ul>
+      <ul id="CLI Options">
```

Comment:
Looks like the space is not allowed here

Reply:
Yes, I updated. But I wonder why `name="Command line usage"` didn't give an error

Reply:
Yes, I updated.